src/components/CommandItem/__tests__/CommandItem.test.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { cleanup, render, waitFor } from '@testing-library/react';
+import '@testing-library/jest-dom';
+
+import CommandItem from '../CommandItem';
+
+afterEach(cleanup); // eslint-disable-line
+
+describe('commandItem', () => {
+  it('should render component with empty entity', () => {
+    const tree = renderer.create(<CommandItem entity={{}} />).toJSON();
+    expect(tree).toMatchInlineSnapshot(`
```

Comment:
Can we keep snapshots in separate files? similar to this - https://github.com/GetStream/stream-chat-react/pull/236/files#diff-9f8593107b8d0b64260cd6c820bef430R39

Reply:
they suggest using inline snapshots whenever the snapshot is short but fine with me if you think it's better to put them in a separate file 
https://github.com/jest-community/eslint-plugin-jest/blob/master/docs/rules/prefer-inline-snapshots.md